tools/ci_build/github/apple/upload_pod_archive_and_update_podspec.sh

```diff
@@ -5,16 +5,25 @@
 set -e
 set -x
 
-USAGE_TEXT="Usage: ${0} <path to pod archive> <path to podspec>"
+USAGE_TEXT="Usage: ${0} <pod archive path pattern (quote to prevent shell expansion)> <podspec path>"
```

Comment:
Some example usage might help to clarify what 'path pattern' means. Not sure what would be considered valid/invalid patterns.